lib/rubocop/cop/style/format_string.rb

```diff
@@ -40,6 +40,50 @@ def message(detected_style)
         def method_name(style_name)
           style_name == :percent ? 'String#%' : style_name
         end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            _, method = *node
```

Comment:
In general we favour names like `_something` over `_` as those are more meaningful. We should probably enable some check for this on our codebase at some point.